libs/@hashintel/ds-components/.gitignore

```
@@ -0,0 +1,2 @@
+# PandaCSS
+styled-system
```

nitpick: Trailing newline
```suggestion
styled-system

```